src/main/java/lotto/domain/LottoShop.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LottoShop {
+    private static final long LOTTO_PRICE = 1000L;
```

Comment:
돈과 같은 단위는 언더바를 통해 가독성을 높일 수 있을 것 같아요! 😄 
```suggestion
    private static final long LOTTO_PRICE = 1_000L;
```

Reply:
아 이게 숫자 어디나 통용되는거였군요... enum 안에서만 가능한건줄 착각하고 있었네요 
꿀팁 감사합니다 😲👍